licenses/JSON11.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2013 Dropbox, Inc.
```

Comment:
Why do we need this license?

Reply:
We use json11 for the profile database IO. The header has the license, but it should be in licenses, too.